src/js/options.js

```diff
@@ -150,6 +158,9 @@ document.addEventListener('DOMContentLoaded', () => {
       fade(popup, 10)
       chrome.storage.sync.clear(() => {
         restoreOptions()
+        chrome.runtime.sendMessage({
```

Comment:
@lepirata Is this similar to console.log()?

Reply:
Never mind I see what it is doing. Nice there is communication between the pop up and the sidebar.